modules/registry-configuring-storage-baremetal.adoc

```diff
@@ -0,0 +1,54 @@
+// Module included in the following assemblies:
+//
+// * architecture/registry
+//* architecture/installation-.adoc
+
+[id="registry-configuring-storage-aws-upi-{context}"]
+= Configuring registry storage for bare metal
+
+During installation, you must configure your registry to use storage.
+
+.Prerequisites
+
+* A cluster on bare metal
+* PV with `ReadWriteMany` access mode
+
+.Procedure
+
+​. Change the `spec.storage.pvc`in the `configs.imageregistry/cluster` resource.
+
+. Provide a suitable persistent volume, such as GlusterFS.
+
+
+////
+. Verify you do not have a registry pod:
++
+----
+$ oc
+----
+
+. Check the registry configuration:
++
+----
+$ oc edit configs.imageregistry.operator.openshift.io
+
+storage:
+---
+
+. Check the `clusteroperator` status:
++
+----
+$
+----
++
+There will be warning similar to:
++
+----
+- lastTransitionTime: 2019-03-26T12:45:46Z
+message: storage backend not configured
+reason: StorageNotConfigured
+status: "True"
+type: Failing
```

Comment:
"Failing" is changed to "Degraded" now:https://github.com/openshift/api/pull/287